ArrowNavigation.js

```diff
@@ -38,27 +52,7 @@ var ArrowNavigation = (function() {
         return value;
     };
 
-    var tolerance_ = 0;
-    var debug_ = false;
-    instance.updateOffsets = function() {
```

Comment:
I did not actually change updateOffsets(). I just moved the keyEventHandler function above updateOffsets() because i removed it from the instance object.